### ISSUE TITLE
incompatibility with vagrant 2.1.3

### ISSUE THREAD

**Issue by y0y0z** [sprotheroe/vagrant-disksize#22]
# vagrant version
```bash
$ vagrant --version
Vagrant 2.1.3
```

# plugin version
```bash
$ vagrant plugin list
vagrant-disksize (0.1.2, global)
```

# Vagrantfile
```bash
$ cat Vagrantfile
Vagrant.configure(2) do |config|
  config.vm.box = 'ubuntu/xenial64'
  config.vagrant.plugins = ['vagrant-disksize']
  config.disksize.size = '50GB'
end
```

# issue
```bash
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

Vagrant:
* Unknown configuration section 'disksize'.
```

i used to work with this amazing plugin for a many years!
right now it does not work and there is no vagrant alternative for this...

can anyone shed the light?

**Comment by spacehorst** [comment]
I confirm the issue

**Comment by proycon** [comment]
Same here on Vagrant 2.1.5

**Comment by 0xC0FFEEC0DE** [comment]
Works for me on 2.1.5. 
Sorry, I don't remember the steps to make it working, try to re-install it or something.

**Comment by sprotheroe** [comment]
Yes, I only see this when installing a new version of Vagrant and the plugins aren't migrated with it. Re-installing the plugin should work (and did for me, on Vagrant 2.2.0) but please let me know if that doesn't work for you and I'll investigate further.

**Comment by jamiejackson** [comment]
I'm trying the plugin for the first time today (in Vagrant 2.1.2). I seem to have needed to install it twice to get it working. :-/

However, once I got it working, it easily made a [tiny Ubuntu box](https://bugs.launchpad.net/cloud-images/+bug/1580596) usable! It resized the disk _and_ the partition with a single LoC.

**Comment by MartinMystikJonas** [comment]
Same issue here
